lib/rules/selector-type-no-unknown/__tests__/index.js

```diff
@@ -132,6 +132,9 @@ testRule(rule, {
     {
       code: ".foo /for/ .bar {}",
       description: "reference combinators"
+    },
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file.